### PR TITLE
Add instructions to README

### PR DESCRIPTION
**Pull request by BradLewis**
Updating the README to include an example image as well as instructions on how to run the app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,49 @@
...
+A <em>blazingly fast</em> web application for generating LastFM collages, written in Go.
+</div>
+
+<br/>
+
+<p align="center">
+  <img alt="SongStitch Logo" src="https://raw.githubusercontent.com/SongStitch/song-stitch/main/collage.png" width="300px"/>
+</p>
+
+### Requirements
```
Comment (TheDen) [inline README.md:25]: Probably should use `## Heading level 2` instead of skipping to `###`

File: README.md [hunk 2]
```diff
@@ -1,15 +1,49 @@
...
 
-[![Website Status](https://img.shields.io/website?style=flat-square&up_message=UP&url=https%3A%2F%2Fsongstitch.art%2F
-)](https://songstitch.art/)
+[![Website Status](https://img.shields.io/website?style=flat-square&up_message=UP&url=https%3A%2F%2Fsongstitch.art%2F)](https://songstitch.art/)
 [![CI status](https://img.shields.io/github/actions/workflow/status/SongStitch/song-stitch/deploy.yml?branch=main&style=flat-square)](https://github.com/SongStitch/song-stitch/actions?query=branch%3Amain)
 [![License](https://img.shields.io/github/license/SongStitch/song-stitch?style=flat-square)](/LICENCE)
 
 </div>
 
+<div align="center">
```
Comment (TheDen) [inline README.md:15]: _noice_

File: README.md [hunk 3]
```diff
@@ -1,15 +1,49 @@
...
+A <em>blazingly fast</em> web application for generating LastFM collages, written in Go.
+</div>
+
+<br/>
+
+<p align="center">
+  <img alt="SongStitch Logo" src="https://raw.githubusercontent.com/SongStitch/song-stitch/main/collage.png" width="300px"/>
```
Comment (TheDen) [inline README.md:22]: This image can't be seen? https://github.com/SongStitch/song-stitch/blob/802d0e30a30a776b9197672693c291ddb7703a0d/README.md

I think once it's in main, you can link to it directly. This is useful if the readme is viewed outside the repo. We can commit the image in `main` (maybe under a `docs/`  folder or just in the root dir), then update the relative link to the hotlink

Would also be good if you run [imageoptim](https://imageoptim.com/mac) on the image with lossless compression + strip metadat

File: README.md [hunk 4]
```diff
@@ -1,15 +1,49 @@
...
+</p>
+
+### Requirements
+
+- Docker
```
Comment (TheDen) [inline README.md:27]: These aren't strict requirements! Should add golang, but not sure on which version.

hmm might be better than adding `go mod` docs etc. though

File: README.md [hunk 5]
```diff
@@ -1,15 +1,49 @@
...
+
+### Usage
+
+1. Clone the repository
+
+```bash
+git clone git@github.com:SongStitch/song-stitch.git
+```
+
+2. Create an API key for [LastFM](https://www.last.fm/api).
```
Comment (TheDen) [inline README.md:38]:
```suggestion
2. Create an API key for [Last.fm](https://www.last.fm/api).
```

File: README.md (hunk 3 above)
Comment (BradLewis) [reply README.md:22]: Yeah it won't be there until this is merged. Tbh I wasn't sure the best way to do that.

File: README.md [hunk 6]
```diff
@@ -1,15 +1,44 @@
...
 
 </div>
 
+<div align="center">
+A <em>blazingly fast</em> web application for generating LastFM collages, written in Go.
+</div>
+
+<br/>
+
+<p align="center">
```
Comment (TheDen) [inline README.md:21]:
```suggestion
<p align="center">
<img alt="SongStitch Logo" src="docs/collage.png" width="300px"/>
 </p>

```

**Review by TheDen — APPROVED** [review]

**Pull request merged**